manager/src/lib.rs

```diff
@@ -199,44 +222,45 @@ impl CronManager {
         fn_deposit: Option<u128>,
         gas_allowance: Option<u64>,
         arguments: Option<Vec<u8>>
-    ) -> Vec<u8> {
+    ) -> Base64VecU8 {
         // TODO: Add asserts to check cadence can be parsed
         log!("cadence {}", &cadence.clone());
         let item = Task {
             owner_id: env::signer_account_id(),
             contract_id,
             function_id,
             cadence,
-            recurring: Some(recurring).unwrap_or(Some(false)).unwrap(),
+            recurring: recurring.unwrap_or(false),
             status: TaskStatus::Ready,
             balance: env::attached_deposit(),
-            fn_deposit: Some(fn_deposit).unwrap_or(Some(0)).unwrap(),
-            gas_allowance: Some(gas_allowance).unwrap_or(Some(GAS_BASE_FEE)).unwrap(),
-            arguments: Some(arguments).unwrap_or(Some(b"".to_vec())).unwrap()
+            fn_deposit: fn_deposit.unwrap_or(0), // for Ⓝ only?
```

Comment:
@mikedotexe correct, since the gas is NEAR, figured its good to keep everything only base level, the logic for all tokens shouldn't be within cron logic